util/test/sub_test

```diff
@@ -142,6 +142,15 @@ import shlex
 import datetime
 import errno
 
+util_dir = os.path.join(os.path.dirname(__file__), '..', '..', 'util')
+config_dir = os.path.join(util_dir, 'config')
+sys.path.insert(0, os.path.abspath(config_dir))
+
+# update PATH to remove CHPL_HOME paths if any
+# (so that prediff etc can use the system python instead of the test venv)
```

Comment:
This comment would be clearer to me if it said something like 

> This essentially deactivates the venv that sub_test is run in, because the test venv is built inside of CHPL_HOME

Reply:
Sure, thanks